core/compose/compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '3.7'
+services:
+  blueos-core:
+    image: bluerobotics/blueos-core:1.2.0-beta.6
```

Comment:
should we use master here? this will get outdated very quickly

Reply:
done